test-forge-blocks.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Hmmm was there a discussion to add this test? I think it will be too heavy and not sure if worth it (in fact failed for this run).
I'd leave this PR to the very minimum which is switching from dapptools to foundry.

Reply:
moved those pieces to a new PR so we can discuss if we want/how we should do it (https://github.com/makerdao/dss-direct-deposit/pull/53)

Reply:
perfect, thanks!